src/main/cpp/include/api/genomicsdb.h

```diff
@@ -44,6 +44,8 @@
 #include <typeinfo>
 #include <vector>
 
+#include "genomicsdb_export_config.pb.h"
```

Comment:
We can't have a dependency to genomicsdb_export_config.pb.h here as this will break Python and Rcpp bindings. Can you move this to genomicsdb.cc?

Reply:
I moved the AnnotationService class definition from genomicsdb.h to genomicsdb.cc so "genomicsdb_export_config.pb.h" is no longer necessary here.